src/main/java/net/dv8tion/jda/api/events/guild/member/update/GuildMemberUpdateAvatarEvent.java

```diff
@@ -70,6 +71,22 @@ public String getOldAvatarUrl() {
         return previous == null ? null : String.format(Member.AVATAR_URL, getMember().getGuild().getId(), getMember().getId(), previous, previous.startsWith("a_") ? "gif" : "png");
     }
 
+    /**
+     * Returns an {@link ImageProxy} for this member's old avatar
+     * <p>
+     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord
```

Comment:
```suggestion
     * Returns an {@link ImageProxy} for this member's old avatar.
     * <p>
     * <b>Note:</b> the old avatar may not always be downloadable as it might have been removed from Discord.
```